# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../ffmpeg/disable-pp_header.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
diff -Nur FFMpeg-20050806.orig/libavcodec/libpostproc/Makefile FFMpeg-20050806/libavcodec/libpostproc/Makefile
--- FFMpeg-20050806.orig/libavcodec/libpostproc/Makefile	2005-06-27 03:55:29.000000000 +0300
+++ FFMpeg-20050806/libavcodec/libpostproc/Makefile	2005-10-24 12:38:22.000000000 +0300
@@ -64,8 +64,8 @@
 	$(LDCONFIG) || true
 endif
 endif
-	mkdir -p $(prefix)/include/postproc
-	install -m 644 $(SRC_PATH)/libavcodec/libpostproc/postprocess.h $(prefix)/include/postproc/postprocess.h
+#	mkdir -p $(prefix)/include/postproc
+#	install -m 644 $(SRC_PATH)/libavcodec/libpostproc/postprocess.h $(prefix)/include/ffmpeg/postprocess.h
 
 
 #
